src/main/java/org/cyclonedx/util/ComponentWrapperDeserializer.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * This file is part of CycloneDX Core (Java).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (c) Steve Springett. All Rights Reserved.
+ */
+package org.cyclonedx.util;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.dataformat.xml.deser.FromXmlParser;
+import org.cyclonedx.model.Ancestors;
+import org.cyclonedx.model.Component;
+import org.cyclonedx.model.ComponentWrapper;
+import org.cyclonedx.model.Descendants;
+import org.cyclonedx.model.Variants;
+
+public class ComponentWrapperDeserializer extends JsonDeserializer<ComponentWrapper>
+{
+  @Override
+  public ComponentWrapper deserialize(
+      final JsonParser parser, final DeserializationContext context)
+      throws IOException, JsonProcessingException
```

Comment:
minor: `JsonProcessingException` is not thrown, could be removed from exceptions list

Reply:
2202b6b